src/stylesheets/components/_mobile-navigation.scss

```diff
@@ -80,8 +80,8 @@
   margin: 0;
   padding: $govuk-spacing-scale-4 $govuk-spacing-scale-4 $govuk-spacing-scale-1 $govuk-spacing-scale-4;
   color: $govuk-grey-1;
-  @include govuk-font-bold-19;
-  text-transform: uppercase;
+  font-size: 19px;
```

Comment:
Could you put your comment about why this is hard fixed in the code, so that this doesn't get 'refactored' away?